collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2783,11 +2782,7 @@ private void displayUIFor(@Nullable BackgroundLocationManager.BackgroundLocation
         // double space where the placeholder is in the text resource. Collapse those spaces.
         // See https://github.com/opendatakit/collect/pull/2864
```

Comment:
Please remove this comment

Reply:
I also ran into a really old comment in SounnerMultiWidget class, please remove this part in the same commit:
> There is a bug in
  android versions previous to 2.0 that affects this widget. You can find the report here:
  http://code.google.com/p/android/issues/detail?id=922 This bug causes text to be white in alert
  boxes, which makes the select options invisible in this widget. For this reason, this widget
  should not be used on phones with android versions lower than 2.0.